common/errors.js

```diff
@@ -136,12 +136,48 @@
         };
     }])
 
-    .factory('$exceptionHandler', ['$log', '$injector' , function($log, $injector) {
-
-        return function(exception, cause) {
-            var ErrorService = $injector.get("ErrorService");
-            ErrorService.handleException(exception);
-        };
-    }])
+    .config(function($provide) {
+        $provide.decorator("$exceptionHandler", ['$log', '$injector' , function($log, $injector) {
+            return function(exception, cause) {
+                var ErrorService = $injector.get("ErrorService");
+                ErrorService.handleException(exception);
+            };
+        }]);
+    });
 
 })();
+
+
+window.onerror = function() {
+    var error = arguments[4];
+    error.stack = [
+        arguments[1],
+        arguments[2],
+        arguments[3]
+    ].join(':');
+    
+    var redirectLink = (chaiseConfig.dataBrowser ? chaiseConfig.dataBrowser : window.location.origin);
+    document.body.innerHTML = '<div modal-render="true" tabindex="-1" role="dialog" class="modal fade in" index="0" animate="animate" modal-animation="true" style="z-index: 1050; display: block;">' 
```

Comment:
It would be better to have this html defined in a template in `common/templates/`. Then you can import it using `$templateCache`.

`var template = $templateCache.get('nameOfTemplate.html');`